test_dtyper.py

```diff
@@ -45,16 +45,13 @@ def bad_command(
 
 
 def test_args():
-    match = r"simple_command\(\) got an unexpected keyword argument 'frog'"
```

Comment:
The reason that I have the separate error messages in that code is to keep everything strict flake8, specifically in line length!

This is where you get a checkmark.

Also, I'm a strong believer in not mixing functionality change code review with cleanups.

I have an alternative, one second.

Reply:
Oh, no, I don't really, the errors are too long. :-D